### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by MancunianSam**
The README had instructions for building the lambda zip manually which is now done by Jenkins. I've replaced it with a description of each Jenkins job and what it does.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,81 +2,28 @@ This is the code and configuration to carry out the antivirus checks on a single
...
 
-Copy the rules file into the lambda directory
+| File name               | Image Name        | Description                                                                                                               |
+|-------------------------|-------------------|---------------------------------------------------------------------------------------------------------------------------|
+| Dockerfile-yara         | yara              | Installs yara and some dependencies it needs like openssl on an alpine image.                                             |
+| Dockerfile-compile      | yara-rules        | Uses yara as the base image. Gets the yara rules from github and compiles them into a single file for yara to use         |
+| Dockerfile-dependencies | yara-dependencies | Installs necessary software on an amazon linux image and zips it up to be used by the lambda                              |
 
-`docker cp rules:/rules/output ./lambda`
+The build job rebuilds all of these images. This isn't necessary most of the time because the dependency versions and yara version don't change that often and so this job is only ever run manually when we need to update the dependencies.
```
Comment (suzannehamilton) [inline README.md:16]: Thanks for adding these explanations! Could you add some deployment instructions as well?

e.g. "To deploy everything, run https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Build/. If you just need to deploy changes to the lambda, run https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Deploy/. (Assuming that's true? And you don't need to run any of the intermediate steps to do the tagging?)

Comment (MancunianSam) [reply README.md:16]: Good point, I've added a section on deploying the images. I've also added in the `--no-cache` and `--pull` arguments to the build commands so we get the latest versions every time.

File: README.md [hunk 2]
```diff
@@ -2,81 +2,42 @@ This is the code and configuration to carry out the antivirus checks on a single
...
-
-Clean up
-
-```
-cd ..
-rm -rf lambda dependencies.zip
-docker stop rules dependencies
-```
+### Python code updates
+This includes our own custom code and any dependencies in requirements.txt This shouldn't need any manual intervention as it should trigger the [test](https://jenkins.tdr-management.nationalarchives.gov.uk/job/TDR%20Antivirus%20Test/) job which in turn runs the bundle and deploy jobs. You can run the test job manually if you need to for any reason.
```
Comment (suzannehamilton) [inline README.md]: Does this only apply to integration? How do you deploy changes to  staging?

Comment (MancunianSam) [reply README.md]: It's just a case of running the deploy job, I've added in a line to say this.

**Review by suzannehamilton — APPROVED** [review]

**Pull request merged**